README.md

```diff
@@ -90,7 +92,48 @@ provider:
     APP_SSM_PARAMETERS: "APP_KEY, DATABASE_URL"
 ```
 
-This will inject `APP_KEY` and `DATABASE_URL` using your service name and stage, for example from `/myapp-staging/APP_KEY`.
+This will resolve `APP_KEY` and `DATABASE_URL` during runtime. Your service name and stage will automatically be prepended to the parameter names. For example, the above example would resolve `/my-app-staging/APP_KEY` and `/my-app-staging/DATABASE_URL`.
+
+Additionally, you can use the [Parameters and Secrets Lambda extension](https://docs.aws.amazon.com/systems-manager/latest/userguide/ps-integration-lambda-extensions.html). If you are resolving a lot of different secrets during runtime, this might improve response times. In order to do so, add the [required layer](https://docs.aws.amazon.com/systems-manager/latest/userguide/ps-integration-lambda-extensions.html#ps-integration-lambda-extensions-add) to your function(s) and ensure you allow `ssm:GetParameter` in your IAM statements.
+
+<details>
+<summary>Minimal serverless.yml example using Parameters and Secrets Lambda extension</summary>
+
+```yml
+service: my-app
+
+provider:
+  name: aws
+  region: eu-central-1
+  stage: staging
+  runtime: provided.al2
+  environment:
+    APP_SSM_PREFIX: /${self:service}-${sls:stage}/
+    APP_SSM_PARAMETERS: "APP_KEY, DB_PASSWORD"
```

Comment:
@georgeboot @tillkruss Passing the parameters by using `Fn::Join` seems to be longer, but cuter?

```yaml
provider:
  environment:
    APP_SSM_PARAMETERS:
      Fn::Join:
        - ","
        - - APP_KEY
          - DB_PASSWORD
```